app/models/pipeline_run.rb

```diff
@@ -386,11 +417,19 @@ def pre_result_monitor?
   def status_display_pre_result_monitor(run_stages)
     # TODO: remove the need for this function by migrating old runs
     state_by_output = {}
-    old_loaders_by_output = { "db_load_host_filtering" => "ercc_counts",
-                              "db_load_alignment" => "taxon_counts",
-                              "db_load_postprocess" => "taxon_byteranges" }
+    # old_loaders_by_output = { "db_load_host_filtering" => "ercc_counts",
```

Comment:
I think you should rebase on current master. Legacy function `status_display_pre_result_monitor` no longer exists ;) so no need to bother with it